### PR TITLE
[NPU] Fix hf generate with save/load generation config for Python (cpp backend)

### PR DESCRIPTION
**Pull request by Oscilloscope98**
## Description

https://github.com/analytics-zoo/nano/issues/1741#issuecomment-2519458756

Fix hf generate with save/load generation config for Python (cpp backend)

### CODE REVIEW

**Comment by jason-dai** [comment]
Do we need to save it for other backends (or even GPU)?

**Comment by Oscilloscope98** [comment]
> Do we need to save it for other backends (or even GPU)?

We already have logic for saving/loading generation config for other backends (NPU Python (acc_lib) and GPU) :)

File: python/llm/src/ipex_llm/transformers/npu_model.py [hunk 1]
```diff
@@ -438,6 +438,12 @@ def load_low_bit(cls, pretrained_model_name_or_path: str, *model_args, **kwargs)
                 model.kv_len = config_dict['kv_len']
                 model.vocab_size = config_dict['vocab_size']
                 model.logits_buffer = torch.empty(1, 1, model.vocab_size, dtype=torch.float32)
+                if model.can_generate():
+                    model.generation_config = GenerationConfig.from_pretrained(
```
Comment (plusbang) [inline python/llm/src/ipex_llm/transformers/npu_model.py:442]: We use try except [here](https://github.com/intel-analytics/ipex-llm/blob/b39d0587555892467831fa8624c4d4c7a68f526f/python/llm/src/ipex_llm/transformers/npu_model.py#L622C9-L630C20), should we use similar operation here?

**Review by plusbang — APPROVED** [review]
Other LGTM

File: python/llm/src/ipex_llm/transformers/npu_model.py (hunk 1 above)
Comment (Oscilloscope98) [reply python/llm/src/ipex_llm/transformers/npu_model.py]: Have updated :)

**Pull request merged**